robocop/checkers/naming.py

```diff
@@ -262,7 +262,6 @@
         msg="'{{ statement_name }}' is deprecated since Robot Framework version "
         "{{ version }}, use '{{ alternative }}' instead",
         severity=RuleSeverity.WARNING,
-        version=">=4.0",
```

Comment:
This rule should not have a restriction on which versions to run because the keywords themselves have a version associated with them in line 645.

Reply:
I agree. Now I see we don't have docs here - should we add more explanation with the list of the deprecated methods/statement and their alternatives per version?

Reply:
It's how it's displayed in docs now:
![image](https://user-images.githubusercontent.com/8532066/162619860-ee6988e7-c43f-4482-8f3b-102d02ac02ec.png)

Reply:
> I agree. Now I see we don't have docs here - should we add more explanation with the list of the deprecated methods/statement and their alternatives per version?

Yes, I think that's a nice idea.